controllers/activemqartemis_controller_test.go

```diff
@@ -122,6 +122,236 @@ var _ = Describe("artemis controller", func() {
 		}
 	})
 
+	Context("broker resource tracking", Label("broker-resource-tracking-context"), func() {
+		It("default user credential secret", func() {
+			By("deploy a broker")
+			brokerCr, createdBrokerCr := DeployCustomBroker(defaultNamespace, nil)
```

Comment:
Could we set custom `adminUser` and `adminPassword` to check their values in the credential secret created by the operator?

Reply:
yes that deserves another test.